target_postgres/target.py

```diff
@@ -79,6 +79,13 @@ def __init__(
             + " ssl_client_certificate or ssl_client_private_key are unset."
         )
 
+        assert self.config.get("add_record_metadata") or not self.config.get(
+            "activate_version"
+        ), (
+            "Activate version messages can't be processed unless add_record_metadata "
+            "is set to true."
```

Comment:
```suggestion
            "is set to true. To ignore Activate version messages instead, Set the `activate_version` configuration to False."
```

Reply:
Steps for how to get the thing to stop failing is always nicest

Reply:
Fixed.